scripts/valhalla_build_extract

```diff
@@ -143,15 +147,16 @@ def get_tiles_with_geojson(all_tile_paths: List[Path], geojson_dir: Path) -> Set
     tile_paths_ = set()
     geojson_polys = get_outer_rings(geojson_dir)
     for tile_path in all_tile_paths:
-        tile_bbox = box(*get_tile_bbox(tile_path))
+        tile_path_id = str(tile_path.relative_to(tiles_dir_))
```

Comment:
this is the important line: just forward the relative tile path to the `get_tile_bbox` function